de.dlr.sc.virsat.model.extension.mechanical/concept/concept.concept

```diff
@@ -0,0 +1,6 @@
+Concept de.dlr.sc.virsat.model.extension.mechanical description "Concept for mechanical engineering" version 0.1

```

Comment:
Style thingy. it should be `Concept de.dlr.sc.virsat.model.extension.mechanical description "Concept for mechanical engineering" version 0.1 {` with the `{` on the same line.